src/variants/lambda/common.mligo

```diff
@@ -0,0 +1,67 @@
+// SPDX-FileCopyrightText: 2022 Tezos Commons
+// SPDX-License-Identifier: LicenseRef-MIT-TC
+
+#if !VARIANT_COMMON
+#define VARIANT_COMMON
+
+#include "../../error_codes.mligo"
+#include "../../helper/unpack.mligo"
+#include "types.mligo"
+#include "../../types.mligo"
+
+let handler_exists(handler_name, ce : string * contract_extra) : bool =
+  Big_map.mem handler_name ce.lambdas
+
+let handler_exists_and_active(handler_name, ce : string * contract_extra) : bool =
+  match Big_map.find_opt handler_name ce.lambdas with
+    | Some (v) -> v.is_active
+    | None -> (failwith proposal_handler_not_found: bool)
```

Comment:
Why not return `false` in this case?

AFAICT this function is only used in `common_proposal_check` where a `false` would end up producing exactly the same result (a `failwith proposal_handler_not_found`).

Moreover, I think that it would be nice to add a documentation comment on this function, as well as the other helpers in this file, to specify what they are supposed to do.

Reply:
>Why not return false in this case?

Seems right. Have changed. Also added some comments.